src/installer/view.rs

```diff
@@ -0,0 +1,1287 @@
+}
+
+pub fn cosigner_key<'a>(
+    key: &str,
+    key_input: &'a mut text_input::State,
+    delete_button: &'a mut Button,
+    warning: bool,
+) -> Element<'a, message::CosignerKey> {
+    let mut col = Column::new().push(
+        Row::new()
+            .push(
+                TextInput::new(key_input, "Key", key, message::CosignerKey::KeyEdited)
+                    .size(15)
+                    .padding(10),
+            )
+            .push(
+                button::transparent(delete_button, Container::new(icon::trash_icon()))
+                    .on_press(message::CosignerKey::Delete),
+            )
+            .spacing(5)
+            .align_items(Align::Center),
+    );
+
+    if warning {
+        col = col.push(text::simple("Please enter a valid key").color(color::WARNING))
+    }
+    Container::new(col.spacing(10)).into()
+}
+
+pub fn define_stakeholder_xpubs_as_stakeholder<'a>(
+    our_xpub: &str,
+    our_xpub_input: &'a mut text_input::State,
+    our_xpub_warning: bool,
+    add_xpub_button: &'a mut Button,
+    other_xpubs: Vec<Element<'a, Message>>,
+    scroll: &'a mut scrollable::State,
+    previous_button: &'a mut Button,
+    save_button: &'a mut Button,
+) -> Element<'a, Message> {
+    let mut col = Column::new()
+        .push(text::bold(text::simple("Your stakeholder xpub:")))
+        .push(
+            TextInput::new(our_xpub_input, "Your stakeholder xpub", our_xpub, |msg| {
+                Message::DefineStakeholderXpubs(message::DefineStakeholderXpubs::OurXpubEdited(msg))
+            })
+            .size(15)
+            .padding(10),
+        )
+        .spacing(10);
+
+    if our_xpub_warning {
+        col = col.push(text::simple("Please enter a valid xpub").color(color::WARNING));
+    }
+
+    layout(
+        scroll,
+        previous_button,
+        Column::new()
+            .push(text::bold(text::simple("Define stakeholders")).size(50))
+            .push(col)
+            .push(
+                Column::new()
+                    .spacing(10)
+                    .push(text::bold(text::simple("Other stakeholders xpubs:")))
+                    .push(Column::with_children(other_xpubs).spacing(10))
+                    .push(
+                        Container::new(
+                            button::white_card_button(
+                                add_xpub_button,
+                                button::button_content(Some(icon::plus_icon()), "Add stakeholder"),
+                            )
+                            .on_press(Message::DefineStakeholderXpubs(
+                                message::DefineStakeholderXpubs::AddXpub,
+                            )),
+                        )
+                        .width(Length::Fill),
+                    ),
+            )
+            .push(
+                Row::new()
+                    .push(
+                        button::primary(save_button, button::button_content(None, "Save"))
```

Comment:
You're not really saving anything, so I don't think there should be a "Save" here. What about "Next"?